.circleci/config.yml

```diff
@@ -122,3 +122,5 @@ workflows:
       - lint-bakery
       - shellcheck/check:
           path: ./scripts
+      - shellcheck/check:
+          path: ./bakery/src/scripts
```

Comment:
Is there any reason why we shouldn't just shellcheck the entire repo (e.g. `path: ./`) at this point in a single job versus creating separate jobs per path? It looks like the only other scripts not currently covered are in `./backend`.

Reply:
Sounds good to me. I guess we figured targeting specific directories would be more efficient, but, this would mean any future scripts would also get scanned without any changes.